### PR TITLE
Switch used-by to mdakits page

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

File: pages/mdakits.md [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+and to advertise these wonderful tools to the MDAnalysis user community,
+we have created an ecosystem of [MDAnalysis Toolkits](https://mdakits.mdanalysis.org/about.html).
 
 
-*If you know of other tools that use MDAnalysis please share
-them with us on [Twitter]({{ site.twitter.url }}) or on our
-[mailinglist]({{ site.mailinglists.discussion.url }}) so that we
-can include them here* or open a pull request by [editing the file `used-by.md`](https://github.com/MDAnalysis/MDAnalysis.github.io/edit/master/pages/used-by.md).
+Please look at the [registry of toolkits](https://mdakits.mdanalysis.org/mdakits.html)
+for a list of toolkits that [meet the minimum MDAKit requirements](insert link here).
```
Comment (IAlibay) [inline pages/mdakits.md:14]: Hold for Oliver's PR on the mdakits registry.

**Comment by IAlibay** [comment]
My jekyll build broke, not sure I'll have time to fix it - if someone wants to check this renders locally that'd be grand.

**Comment by lilyminium** [comment]
Screenshot 2023-09-23 at 2 38 54 pm

I checked the render -- it looks good! (Slightly inaccurate/ill-proportioned as I zoomed out for the screenshot).

File: pages/mdakits.md [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
 
-While our documentation is mostly focused on using MDAnalysis for
-exploratory analysis it is equally well suited to build your own
-analysis library on top of it. Below is a list of projects we know
-about that use MDAnalysis.
+MDAnalysis is developed with extensibility in mind, allowing
+scientists to create various tools with the components
+that the library provides. To help developers in this process,
+and to advertise these wonderful tools to the MDAnalysis user community,
+we have created an ecosystem of [MDAnalysis Toolkits](https://mdakits.mdanalysis.org/about.html).
```
Comment (lilyminium) [inline pages/mdakits.md:10]:
```suggestion
we have created an ecosystem of [MDAnalysis Toolkits, or "MDAKits"](https://mdakits.mdanalysis.org/about.html).
```

File: pages/mdakits.md [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
 
 
-*If you know of other tools that use MDAnalysis please share
-them with us on [Twitter]({{ site.twitter.url }}) or on our
-[mailinglist]({{ site.mailinglists.discussion.url }}) so that we
-can include them here* or open a pull request by [editing the file `used-by.md`](https://github.com/MDAnalysis/MDAnalysis.github.io/edit/master/pages/used-by.md).
+Please look at the [registry of toolkits](https://mdakits.mdanalysis.org/mdakits.html)
+for a list of toolkits that [meet the minimum MDAKit requirements](insert link here).
+These are regularly tested against the latest and development versions of
+MDAnalysis, so if you see a green badge it should be ok to use!
```
Comment (lilyminium) [inline pages/mdakits.md:16]:
```suggestion
MDAnalysis, so if you see a green badge, we expect that it should be good to use!
```

"Ok" felt a little unenthusiastic -- minor comment, but swap for "good"?

File: pages/mdakits.md [hunk 4]
```diff
@@ -1,21 +1,32 @@
...
 
-## Visualization tools
+*If you have your own MDAKit and you would like to add it to the
+registry, please look at the [contribution instructions](https://mdakits.mdanalysis.org/add.html)
+or share it with on our [mailing list]({{ site.mailinglists.discussion.url }})!*
+
+
+## Other (non-MDAKit registered) tools using MDAnalysis
+
+Below is a list of non-MDAKit registered projects which we know to use MDAnalysis:
```
Comment (lilyminium) [inline pages/mdakits.md:26]:
```suggestion
Below we list projects that use MDAnalysis and are not registered as MDAKits:
```

Stylistic suggestion

**Review by lilyminium — DISMISSED** [review]
LGTM but blocking so it's not accidentally merged before updating the link to the adding an mdakit page -- feel free to dismiss my review when that happens!

File: pages/mdakits.md (hunk 3 above)
Comment (IAlibay) [reply pages/mdakits.md:16]: 🤔 We probably should be even clearer that it's not on us but on the MDAKit authors, I might try something else.

Comment (IAlibay) [reply pages/mdakits.md]: Updated

File: pages/mdakits.md [hunk 5]
```diff
@@ -1,21 +1,32 @@
...
+that the library provides. To help developers in this process,
+and to advertise these wonderful tools to the MDAnalysis user community,
+we have created an ecosystem of [MDAnalysis Toolkits, or "MDAKits"](https://mdakits.mdanalysis.org/about.html).
 
 
-*If you know of other tools that use MDAnalysis please share
-them with us on [Twitter]({{ site.twitter.url }}) or on our
-[mailinglist]({{ site.mailinglists.discussion.url }}) so that we
-can include them here* or open a pull request by [editing the file `used-by.md`](https://github.com/MDAnalysis/MDAnalysis.github.io/edit/master/pages/used-by.md).
+Please look at the [registry of toolkits](https://mdakits.mdanalysis.org/mdakits.html)
```
Comment (RMeli) [inline pages/mdakits.md:13]:
```suggestion
Please look at the [registry of MDAKits](https://mdakits.mdanalysis.org/mdakits.html)
```

**Pull request merged**